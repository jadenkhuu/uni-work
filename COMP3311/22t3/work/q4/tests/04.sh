psql racing -c "select * from q4('neigh') order by horse;"
