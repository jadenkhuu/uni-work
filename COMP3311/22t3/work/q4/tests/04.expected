           horse           | average 
---------------------------+---------
 Always Neighing           |   11133
 Arnold Schwarze-neigh-ger |   27052
 Hay Neighbor              |   75000
 Hermoineigh               |    8947
 Liam Neighson             |  108538
 Neigh Sayer               |   77538
 NeighNeigh Leaks          |   65000
 Ron Neighsly              |     833
 Sir Neighs Alot           |   56250
(9 rows)
